src/cursor/abstract_cursor.ts

```diff
@@ -344,7 +345,9 @@ export abstract class AbstractCursor<
 
   /** Get the next available document from the cursor, returns null if no more documents are available. */
   next(): Promise<TSchema | null>;
+  /** @deprecated Callbacks are deprecated and will be removed in the next major version */
   next(callback: Callback<TSchema | null>): void;
+  /** @deprecated Callbacks are deprecated and will be removed in the next major version */
```

Comment:
```suggestion
```

Reply:
Let's save type changes for another PR/ticket

Reply:
https://jira.mongodb.org/browse/NODE-4623